src/txn/txn_recover.c

```diff
@@ -822,25 +823,95 @@ __recovery_file_scan_prefix(WT_RECOVERY *r, const char *prefix, const char *igno
         if (ignore_suffix != NULL && WT_SUFFIX_MATCH(uri, ignore_suffix))
             continue;
         WT_RET(c->get_value(c, &config));
-        WT_RET(__recovery_setup_file(r, uri, config));
+        WT_RET(__recovery_func(r, uri, config));
     }
     WT_RET_NOTFOUND_OK(ret);
     return (0);
 }
 
+/*
+ * __metadata_clean_incomplete_table --
+ *     For each table metadata entry, check that the table was fully created. If not, clean up the
+ *     incomplete table.
+ */
+static int
+__metadata_clean_incomplete_table(WT_RECOVERY *r, const char *uri, const char *config)
+{
+    WT_DECL_ITEM(buf);
+    WT_DECL_RET;
+    WT_TABLE *fake_table_handle;
+    const char *cfg[] = {config, NULL};
+    const char *drop_cfg[] = {WT_CONFIG_BASE(r->session, WT_SESSION_drop), "force=true", NULL};
+    const char *format_name;
```

Comment:
np: AFAIS in all other functions it called simply `name` and not `format_name`


```suggestion
    const char *name;
```